### DISCUSSION TITLE
I don't get an api key with deepl free.

### DISCUSSION THREAD

**Discussion by mzso** [sienori/simple-translate#396] [Q&A]
I created an account then clicked the link in step two, but I get redirected to: https://www.deepl.com/account/summary
And I only have these options:
![kép](https://user-images.githubusercontent.com/4479090/175770073-413816be-0bec-475b-9217-7b129fed217d.png)
(Of course deepl want to subscribe me for pro with credit card data and personal information, but I definitely don't want that.)

**Comment by MOULINIER** [comment]
You have to put your credentials for the DeepL API Free plan, personnaly I put a single use card here
